src/index.js

```diff
@@ -18,6 +18,7 @@ import ProgressBar from './elements/ProgressBar/ProgressBar'
 import Rating from './elements/Rating/Rating'
 import SelectField from './elements/SelectField/SelectField'
 import Shield from './elements/Shield/Shield'
+import Slider from './elements/Slider/Slider'
```

Comment:
`DoubleSlider` should entire replace `Slider` if I understand correctly that `DoubleSlider` supersedes `Slider`.